### PR TITLE
Not for merge. Just testing.

### PR DESCRIPTION
**Pull request by niklasborwell**

### CODE REVIEW

**Comment by niklasborwell** [comment]
Sorted vector list seems to fail for me. Is there an issue with this class?

**Comment by marcusnaslund** [comment]
Not that I know of.

**Comment by marcusnaslund** [comment]
It has never failed on `develop`...

**Comment by niklasborwell** [comment]
@emilwestergren You've done some changes in VectorList in 1.4. See anything suspicious?

File: test/collections/SortedVectorListTest.ooc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+use unit
+
+SortedVectorListClassTester: class {
+	_value: Float
+	value ::= this _value
+	init: func (=_value)
+}
+
+sortedVectorListIntComparator := func (a, b: Int*) -> Bool { a@ > b@ }
+sortedVectorListClassComparator := func (a, b: SortedVectorListClassTester*) -> Bool { a@ value > b@ value }
```
Comment (marcusnaslund) [inline test/collections/SortedVectorListTest.ooc]: @niklasborwell The problem is here. Change both `>` to `<`, and the test goes through.
The reason is that, in `primaryFeature1.4`, `VectorList sort()` has changed its argument from expecting a `greaterThan` comparator to an `isLess` comparator, in order to be consistent with other sorting-related methods.

**Comment by marcusnaslund** [comment]
All kean tests passed now.

**Pull request merged**